mi/reporting/report.py

```diff
@@ -0,0 +1,38 @@
+from typing import Generator
+
+import fire
+
+from helpers.aws_session import new_session_from_env
+from mi.reporting.actions import each_query_event, perform_query, write_csv
+
+
+def make_reports(
```

Comment:
For the future, but will we need the ability to run a specific report rather than all the reports in the future?

Reply:
yeah maybe - i think we need to see how this is used before we can spec it further